src/main/kotlin/Car.kt

```diff
@@ -0,0 +1,42 @@
+import com.univocity.parsers.csv.CsvParser
+import com.univocity.parsers.csv.CsvParserSettings
+
+class Car() {
+
+
+    fun interestOfDriving(numberOfCars: Int): MutableList<Int> {
+        var ListOfInterestToDrive: MutableList<Int> = mutableListOf()
+        var interestToDrive = 0
```

Comment:
it doesn't really matter what type it is, does it? In the end all that is done with that information is if (this) then that. So whether it's true or a 1, or false and a 0.